testsuite/classlibrary/TestServer_boot.sc

```diff
@@ -12,10 +12,11 @@ TestServer_boot : UnitTest {
 
 	cycleNotify { |server|
 		// XXX: no efficient way to wait
+		// FIXME: supernova hangs here unless we resend notify in the while loop
 		server.notify_(false);
-		while { server.notified } { 0.1.wait };
+		while { server.notified } { server.notify_(false); 0.1.wait };
 		server.notify_(true);
-		while { server.notified.not } { 0.1.wait };
+		while { server.notified.not } { server.notify_(false); 0.1.wait };
```

Comment:
for the sake of clarity, it would probably be better here to write `statusWatcher.sendNotifyRequest(false)`. Then the code immediately makes sense with the comment.

Reply:
done